packages/abi-typegen/package.json

```diff
@@ -43,10 +43,9 @@
     "dist"
   ],
   "scripts": {
-    "pretest": "run-s build:forc-callpaths build:forc",
+    "pretest": "pnpm build:forc",
     "build": "tsup",
     "build:forc": "pnpm fuels-forc build -p test/fixtures/forc-projects --release",
-    "build:forc-callpaths": "pnpm fuels-forc build -p test/fixtures/forc-projects --json-abi-with-callpaths",
```

Comment:
The `--json-abi-with-callpaths` flag is now the default so this isn't necessary anymore.